Neos.Neos/Classes/Controller/LoginController.php

```diff
@@ -120,10 +139,20 @@ public function indexAction($username = null, $unauthorized = false)
         }
         $currentDomain = $this->domainRepository->findOneByActiveRequest();
         $currentSite = $currentDomain !== null ? $currentDomain->getSite() : $this->siteRepository->findDefault();
+
+        $trustedPropertiesToken = $this->mvcPropertyMappingConfigurationService->generateTrustedPropertiesToken([
+            'lastVisitedNode',
+            '__authentication[Neos][Flow][Security][Authentication][Token][UsernamePassword][username]',
+            '__authentication[Neos][Flow][Security][Authentication][Token][UsernamePassword][password]',
+        ]);
+
```

Comment:
By using the Neos.Fusion.Form:Form form the new neos/fusion-form this should not be necessary. This would add the dependency to neos/fusion-form to neos/neos which we are planning to do anyways.

Reply:
Ok, was not sure if we want that already.
But if it's only a plan should I really do that now?